src/collections/Breadcrumb/Breadcrumb.js

```diff
@@ -34,35 +34,17 @@ function Breadcrumb(props) {
   const rest = getUnhandledProps(Breadcrumb, props)
   const ElementType = getElementType(Breadcrumb, props)
 
-  if (!_.isNil(children)) {
-    return <ElementType {...rest} className={classes}>{children}</ElementType>
-  }
+  if (!_.isNil(children)) return <ElementType {...rest} className={classes}>{children}</ElementType>
 
   const childElements = []
 
   _.each(sections, (section, index) => {
     // section
-    const breadcrumbSection = BreadcrumbSection.create(section)
-    childElements.push(breadcrumbSection)
+    childElements.push(BreadcrumbSection.create(section))
 
     // divider
     if (index !== sections.length - 1) {
-      // TODO generate a key from breadcrumbSection.props once this is merged:
-      // https://github.com/Semantic-Org/Semantic-UI-React/pull/645
-      //
-      // Stringify the props of the section as the divider key.
-      //
-      // Section:     { content: 'Home', link: true, onClick: handleClick }
-      // Divider key: content=Home|link=true|onClick=handleClick
-      let key
-      if (section.key) {
-        key = `${section.key}_divider`
-      } else {
-        key = _.map(breadcrumbSection.props, (v, k) => {
-          return `${k}=${typeof v === 'function' ? v.name || 'func' : v}`
-        }).join('|')
-      }
-      childElements.push(BreadcrumbDivider.create({ content: divider, icon, key }))
```

Comment:
Darn, this is technically a breaking change.  Users who are using objects to define a breadcrumb and also not defining a key will now need to add the key.  Otherwise, their breadcrumbs won't render.